final_prep/mod_zero_hero.rb

```diff
@@ -2,40 +2,78 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = "Hobbes"
+special_ability = "ability to eat an ungodly amount of oranges"
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "Hello citizens! I am #{hero_name}!"
+catchphrase = "Did you know? I have the #{special_ability}"
+
 # Declare two variables - power AND energy - set to integers
 
+power = 1
+energy = 2
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy + 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
+arch_enemies = ["The United States Government of America", "That One Old Lady Who Calls The Cops For No Reason", "Toby the Rat"]
+sidekicks = ["Limes", "Oranges", "Josh (he's pretty cool)"]
+
 # Print the first sidekick to your terminal
 
+puts sidekicks[0]
+
 # Print the last arch_enemy to the terminal
 
+puts arch_enemies[2]
```

Comment:
What's an alternative way to print the last `arch_enemy` if you didn't know the length of the array?